internal/controller/certificates/policies/checks.go

```diff
@@ -303,6 +321,12 @@ func SecretTemplateMismatchesSecretManagedFields(fieldManager string) Func {
 			})
 		}
 
+		// Remove the base Annotations from the managed Annotations so we can compare
+		// 1 to 1 against the SecretTemplate.
+		for k := range baseAnnotations {
```

Comment:
Question: should a base annotation being mismatched not be a violation? Say, in a case where a new alt name has been added to the cert and the alt names annotation needs updating. Or, would check be better done elsewhere/is already done elsewhere?

Reply:
Indeed, this is a violation of the trigger controller, rather than the post issuance policy chain.

Reply:
This is the check: https://github.com/jetstack/cert-manager/pull/4746/files#diff-65dfe99e2897856c2b1742247d25d7d285612dd2f71252513b9b2ff457841d3aR94

Reply:
I think we should be adding a policy check to ensure that those annotations are correct and kept up to date :+1: 

We do not currently do this. I will add this in a follow up PR.